### DISCUSSION TITLE
Minimized program

### DISCUSSION THREAD

**Discussion by Mephisto090** [cefsharp/CefSharp.Dom#25] [Q&A]
When the program is minimized, `ClickAsync` stops working. If it's in the background it's no problem. I think it's better that it still works even if it's minimized

**Comment by amaitland** [comment]
That's the expected behaviour for a non headless browser.  

In the context of this project it's not something I can control.

**Reply by Mephisto090** [reply]
Can you think of a way to somehow bypass this in my program and still minimize the program, which it actually doesn't do?

**Reply by Mephisto090** [reply]
Do you think making invisible would work?

**Reply by oguzhankrcb** [reply]
Is there any progress about it, i am having the same issue and i want to know if there is any solution for this problem.

**Reply by amaitland** [reply]
Keeping the browser active whilst minimized is possible, for WinForms try https://github.com/cefsharp/CefSharp/discussions/4202#discussioncomment-3394347

WPF look at the ChromiumWebBrowser source for calls to WasHidden
